### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by bythewaters**

### CODE REVIEW

File: .idea/.gitignore
Comment (ArsenPidhoretskyi) [inline .idea/.gitignore:1]: This file must be not into root

File: city/models.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+from database import Base
+
+
+class City(Base):
+    __tablename__ = "city"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String(255), nullable=False)
+    additional_info = Column(String(511))
+
```
Comment (ArsenPidhoretskyi) [inline city/models.py]: Empty line

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]
Check all files for empty line

File: .DS_Store
Comment (EdAlekseiev) [inline .DS_Store:1]: Delete this file from your commit and add this file to .gitignore

File: city/crud.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
+from typing import Optional, Any, Sequence
+from sqlalchemy import select, Row, RowMapping, insert, update, delete
+from sqlalchemy.ext.asyncio import AsyncSession
+from city import models, schemas
+
+
+async def get_all_cities(
+        db: AsyncSession
+) -> Sequence[Row | RowMapping | Any]:
```
Comment (EdAlekseiev) [inline city/crud.py]: Improve your typehint for return value

Comment (EdAlekseiev) [inline city/crud.py]:
```suggestion
) -> Sequence[City]:
```

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]

**Review by YukihiroSM — APPROVED** [review]
Looks good to me!

**Pull request closed without merging**